example/sparse/wide_deep/train.py

```diff
@@ -17,8 +17,8 @@
 
 import mxnet as mx
 from mxnet.test_utils import *
-from data import *
-from model import *
+import data
```

Comment:
suggested - can we import specific methods.

Reply:
Thanks @sandeep-krishnamurthy. Made the change